### PR TITLE
Enable IPython autocomplete of header keys.

### LINKED ISSUE #114: Enable ipython completion of header keys?
Might be nice for headers at least. See https://github.com/astropy/astropy/pull/7071 for implementation in astropy. No hurry, though. cc @cczhu.

### PR DESCRIPTION
**Pull request by cczhu**
Fixes #114.

### CODE REVIEW

File: baseband/tests/coveragerc [hunk 1]
```diff
@@ -28,4 +28,7 @@ exclude_lines =
    def main\(.*\):
 
    # Ignore branches that don't pertain to this version of Python
-   pragma: py{ignore_python_version}
\ No newline at end of file
+   pragma: py{ignore_python_version}
+
+   # Don't complain about IPython completion helper
+   def _ipython_key_completions_
```
Comment (mhvk) [inline baseband/tests/coveragerc]: missing eol

**Comment by cczhu** [comment]
Works for frames and framesets now too.

File: baseband/tests/coveragerc (hunk 1 above)
Comment (cczhu) [reply baseband/tests/coveragerc]: It actually didn't have one before.  Not sure if it's auto-eliminated; I tried adding one, and it gets deleted.

Comment (mhvk) [reply baseband/tests/coveragerc]: It's there now!

**Pull request merged**